classes/Access.php

```diff
@@ -304,15 +322,16 @@ public function removeModuleAccess($idProfile, $idRole)
     /**
      * Update legacy access
      *
-     * @param int    $idProfile Profile ID
-     * @param int    $idTab     Tab ID
-     * @param string $lgcAuth   Legacy authorization
-     * @param int    $enabled   Whether access should be granted
+     * @param int    $idProfile     Profile ID
+     * @param int    $idTab         Tab ID
+     * @param string $lgcAuth       Legacy authorization
+     * @param int    $enabled       Whether access should be granted
+     * @param int    $addFromParent Child from parents
      *
      * @return string Whether legacy access has been successfully updated ("ok", "error")
      * @throws Exception
      */
-    public function updateLgcAccess($idProfile, $idTab, $lgcAuth, $enabled)
+    public function updateLgcAccess($idProfile, $idTab, $lgcAuth, $enabled, $addFromParent)
```

Comment:
Hello @kompilorb,
Can you add a default parameters please ?! in order to keep retro-compat!